geometry_msgs/PoseStamped[]  pred_traj
mpc_msgs/obstacle_gmm[] obstacles
---
float64 estimated_risk
